CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#100](https://github.com/nf-core/proteinfold/pull/100) Update containers for AlphaFold2 and ColabFold local modules
 
+- [#105](https://github.com/nf-core/proteinfold/pull/105) Update COLABFOLD_BATCH docker container, metro map figure and nextflow schema description
```

Comment:
Reference #104 and maybe #102? Also might be better for next times to do separates PRs that reference each issue